test/system/src/test/java/io/pravega/test/system/AbstractKVSFailoverTest.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Instead of creating a new class for this, my question is: can we extend `AbstractReadWriteTest` to support writes/reads to KVS (current it supports regular writers and transactional writers). Then, create a new method in `AbstractFailoverTests` that is something like `performFailoverForTestsInvolvingKVS()` that is finally invoked by the `KvsWithFailOverTest.java`? It would be great to stick to the existing structure, so we do not have repeated logic for every king of API we develop. As you can infer, the fact of writing and reading from KVS is a separate concern to that of inducing failovers, so we can implement only the KVS part and reuse as much as possible whatever logic is already there for failovers.

Reply:
Removed AbstractKVSFailoverTest.java class, and as suggested performFailoverForTestsInvolvingKVS() in AbstractFailoverTests class file and KVS function added in AbstractReadWriteTest class